### PR TITLE
feat: update localizable

### PR DESCRIPTION
**Pull request by zhouxl**
## Related Issue

#546 #545 #552 #554 #570 #544 #540 

## Need Regression Testing

- [x] Yes
- [ ] No

## Risk Assessment

- [ ] Low
- [x] Medium
- [ ] High

### CODE REVIEW

**Review by lmcmz — APPROVED** [review]

File: FRW/Resource/en.lproj/Localizable.strings
Comment (jeden) [inline FRW/Resource/en.lproj/Localizable.strings:1]: At line 485 this doesn't seem to have been updated (ticket #544):

```
"no_account_found_msg" = "No results. Please check your input and try again.";
```

**Review by jeden — CHANGES_REQUESTED** [review]
I can't find the changes related to #540. Are these already implemented, just missing the language translations?

File: FRW/Resource/en.lproj/Localizable.strings
Comment (zhouxl) [reply FRW/Resource/en.lproj/Localizable.strings:1]: fixed

**Comment by zhouxl** [comment]
> I can't find the changes related to #540. Are these already implemented, just missing the language translations?

yes, Let me go to crowdin and check it out.

**Review by hellolighten-team — APPROVED** [review]

**Review by jeden — APPROVED** [review]

**Pull request merged**